netutils/interface.py

```diff
@@ -352,3 +428,94 @@ def sort_interface_list(interfaces: t.List[str]) -> t.List[str]:
     for ifname in interfaces:
         _insert_nodes(root, _split_interface_tuple(ifname))
     return list(iter_tree(root, []))
+
+
+INTERFACE_LIST_ORDERING_OPTIONS = {"alphabetical": sort_interface_list}
+
+
+def abbreviated_interface_name_list(  # pylint: disable=R0913, R0914
+    interfaces, addl_name_map=None, addl_reverse_map=None, verify=False, order=None, reverse=None
+):
+    """Function to return a list of interface's abbreviated name.
+
+    Args:
+        interfaces (list): List of interface names you are attempting to abbreviate.
+        addl_name_map (dict, optional): A dict containing key/value pairs that updates the base mapping. Used if an OS has specific differences. e.g. {"Po": "PortChannel"} vs {"Po": "Port-Channel"}. Defaults to None.
+        addl_reverse_map (dict, optional): A dict containing key/value pairs that updates the abbreviated mapping. Defaults to None.
+        verify (bool, optional): Whether or not to verify the interface matches a known interface standard. Defaults to False.
+        order (str, optional): Determines what order the list of interfaces should be returned in. Defaults to None.
```

Comment:
It seems like the default would fail.

Reply:
As in `order` being set to `None`? L470-471 only check that the order specified is valid if its defined and L502-503 only attempt to order the list if it's defined.